File: Daily Coding Problem/day5.js
function removeDuplicates(arr) {
    let dic = {} 
    for (let i =0;i<arr.length;i++) {
        if (dic[arr[i]]){
            arr.splice(i,1)
            i--
        }
        else {
            dic[arr[i]] = 1 
        }
    }
    return arr
}

let arr = [1,2,2,2,2,3,3,3,3,3,5,5]
console.log(removeDuplicates(arr))

File: Daily Coding Problem/day5.py
 
def remove_duplicate(a):
    dic = {} 
    i = 0
    while (i<len(a)):
        print(i)
        if (a[i] in dic):
            del a[i]
            i = i-1
        else:
            dic[a[i]] = 1 
        i = i+1
    return a
print(remove_duplicate([1,2,2,3,3,3,3,5,5,5]))